src/utils/index.ts

```diff
@@ -254,3 +254,12 @@ export function cleanUndefined(obj: Object) {
     {}
   );
 }
+
+// converts named colors (red, antiquewhite, etc) to hex
+export function standardizeColor(color: string | undefined) {
```

Comment:
Can use `Maybe<string>` here, since we don't differentiate between null and undefined.